### ISSUE TITLE
Removing first frame crashes application

### ISSUE THREAD

**Issue by CandyFace** [pencil2d/pencil#639]
### --Issue Summary--
If you make x frames and draw at the last frame, then moves the red bar to the first frame and try to remove the frame, you'll see that see application either hangs or crashes.

### --Actual Results--

Application crashes

### --Expected Results--

Frame should be removed successfully.

### --Video or Image Reference--
![crash](https://cloud.githubusercontent.com/assets/1045397/24919967/bc54c1b8-1ee5-11e7-84e8-173098edb760.gif)

### --Steps to reproduce--
1. create x number of frames
2.  draw something on the last frame
3. move the red bar to the first frame and remove the frame.

### --System Information--
+ Pencil2D Version:
Pencil2D nightly build April 11 2016.

+ Operating System:
Mac OS X 10.11.6

+ RAM Size:
8 GB 1600 MHz DDR3

+ CPU:
Processor: 1.7 GHz Intel Core i7

+ Graphics Tablet:
Intel HD 5000

---

```C++
void Editor::removeKey()
{
	Layer* layer = layers()->currentLayer();
    if ( layer != NULL)
    {
        layer->removeKeyFrame( currentFrame() );
		
        scrubBackward();
        mScribbleArea->updateCurrentFrame();
	}
    Q_EMIT layers()->currentLayerChanged( layers()->currentLayerIndex() ); // trigger timeline repaint.
}
```

**Comment by CandyFace** [comment]
It doesn't happen consistently. I suspect that it's a value that doesn't get initialized properly.

**Comment by CandyFace** [comment]
This should be fixed now in PR #646
